src/DonationForms/resources/registrars/templates/fields/Gateways.tsx

```diff
@@ -142,12 +144,26 @@ export default function Gateways({isTestMode, defaultValue, inputProps, gateways
 /**
  * @since 3.0.0
  */
-function GatewayOption({gateway, defaultChecked, inputProps}: GatewayOptionProps) {
+function GatewayOption({gateway, defaultChecked, inputProps, isActive}: GatewayOptionProps) {
+    const gatewayClass = `givewp-fields-gateways__gateway--${gateway.id}`;
+    const className = cx('givewp-fields-gateways__gateway', {
+        [gatewayClass]: true,
+        'givewp-fields-gateways__gateway--active': isActive,
+    });
+
     return (
-        <li>
-            <input type="radio" value={gateway.id} id={gateway.id} defaultChecked={defaultChecked} {...inputProps} />
-            <label htmlFor={gateway.id}> {sprintf(__('Donate with %s', 'give'), gateway.label)}</label>
-            <div className="givewp-fields-payment-gateway">
+        <li className={className}>
+            <label>
+                <input
```

Comment:
Since the input was moved inside the label, we're missing some gateway styles for the gateway icons.  This is still in the new designs. We'll have to come up with a different way to accomplish this:
 
<img width="769" alt="Screenshot 2023-08-31 at 3 07 06 PM" src="https://github.com/impress-org/givewp/assets/10138447/f4bdd075-b9a8-4111-be8c-4fcf6a0023e9">

Reply:
Good catch! I refactored it a bit since we'll ultimately want to allow gateways to set their icons, also I'm more confident in styling this markup. In 699974f16